tests/39-basic-api_level.c

```diff
@@ -40,5 +41,20 @@ int main(int argc, char *argv[])
 	if (api != 1)
 		return -3;
 
+	rc = seccomp_api_set(max_api);
```

Comment:
See the comment above, let's drop this check.

As odd as it may sound, I'm okay with putting in explicit checks for each supported API level as the API level is added to the library.

Reply:
I should have left a comment in the code with my intentions but I was thinking that this would force anyone adding a new API level to add a set/get test for it.

Reply:
Yes, I understand, and generally applaud tests like that, but I'm thinking that any new API level addition is going to require additional tests anyway so it isn't as large a concern.  I guess I could be convinced that this is the Right Thing To Do if you feel strongly about it.

Reply:
I don't feel strongly about it. I just wanted to make sure you knew what I was thinking when I wrote the tests since I did a poor job of recording what I was thinking. I'll get this PR updated hopefully by EOD. Thanks!